spec/feature/user_manage_groups_spec.rb

```diff
@@ -0,0 +1,81 @@
+require 'rails_helper'
+
+RSpec.feature 'Managing groups' do
+  scenario 'users can create a group' do
+    feature_login
+    group = build(:group)
+
+    click_link 'Create Group'
```

Comment:
Strange that you don't have to click on the plus sign first and this works. Do you know why that is?